Dockerfile

```diff
@@ -81,18 +93,26 @@ RUN git clone https://github.com/OCA/OCB.git -b 10.0 odoo && \
 ADD sources/odoo.conf /opt/odoo/etc/odoo.conf
 ADD auto_addons /opt/odoo/auto_addons
 
-RUN git config --global user.email "contact@elico-corp.com"
-RUN git config --global user.name "Elico Corp Odoo Docker"
+User 0
+
+# Provide read/write access to group (for host user mapping). For some reason, the
+# files added in a volume (e.g. odoo.conf) belong to root. This command imperatively
+# needs to run before creating the volumes.
+RUN chmod -R 775 /opt/odoo && chown -R odoo:odoo /opt/odoo
```

Comment:
This command will take a lot of time and maybe it could be merged to line 85.
* `chmod` could be replaced by `mkdir -m`
* `chown` is unnecessary according to comments above line85